### DISCUSSION TITLE
Welcome to Notebooks Discussions!

### DISCUSSION THREAD

**Discussion by yammesicka** [PythonFreeCourse/Notebooks#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Pickacho** [comment]
Hi Yam 
will you update the repo to include the other parts of the course? through week 16?

**Reply by yammesicka** [reply]
Hi :)

There are no such plans for 2021.
In the last course, most of the learning in weeks 9-16 was according to the syllabus and wasn't organized in notebooks.
Creating notebooks for a single chapter takes me around 1 month of intensive work, and unfortunately I can't see myself dedicating any more time for this project in the upcoming year.

**Reply by Pickacho** [reply]
Hi Yam,
Thank you for taking the time to answer.
I know that others and I felt the enormous amount of investment that you and your team provided to the course during the course.
 I think that most of the real-world learning was in weeks 9 -16. 
and without real reference in the syllabus - when a new person will come -> lots of the course materials are lost to him/
I believe that adding weeks 9-16 will give great value to all.
Thanks in advance
Ran

**Comment by yoelnavon** [comment]
hi
how to install on iOS ?
